src/coredata.cs

```diff
@@ -127,6 +137,9 @@ public interface NSEntityDescription : NSCoding, NSCopying {
 		[Export ("managedObjectClassName")]
 		string ManagedObjectClassName { get; set; }
 
+		[Export ("renamingIdentifier")]
+		string RenamingIdentifier { get; set; }
```

Comment:
This is cool as long as they don't do string pointer comparisons (it'd have to be NSString if so).